src/emitters/photon_emitter_old.cpp

```diff
@@ -210,4 +210,4 @@ class PhotonEmitter final : public Emitter<Float, Spectrum> {
 
 MI_IMPLEMENT_CLASS_VARIANT(PhotonEmitter, Emitter)
 MI_EXPORT_PLUGIN(PhotonEmitter, "Photon emitter")
-NAMESPACE_END(mitsuba)
\ No newline at end of file
+NAMESPACE_END(mitsuba)
```

Comment:
Not sure what's happened here - check the newline at the end of this file.

Reply:
Every other file like this in this directory has a newline at the end of the file, so keeping this in here for consistency.